Author: aszlig <aszlig@nix.build>
Date:   Fri Nov 30 06:10:22 2018 +0100

    Pin node-sass version to <= 4.10.0
    
    Since node-sass version 4.11.0, importing CSS files doesn't work anymore
    and it will throw an import error.
    
    In our case we import node_modules/bootstrap-vue/dist/bootstrap-vue,
    which doesn't include SASS files and only CSS files, so importing will
    fail.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: node-sass-4.10.0.patch

diff --git a/package.json b/package.json
index 882a3b699..04a96af4c 100644
--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
     "mongoose": "^5.3.4",
     "morgan": "^1.7.0",
     "nconf": "^0.10.0",
-    "node-sass": "^4.9.0",
+    "node-sass": "<=4.10.0",
     "ora": "^3.0.0",
     "pageres": "^4.1.1",
     "passport": "^0.4.0",
